v0/destinations/attribution/transform.js

```diff
@@ -40,6 +40,7 @@ function getTransformedJSON(message) {
     ? message.properties
```

Comment:
Here as well you can just do `get(message, "properties") it will return undefined if `properties` is not present